### PR TITLE
[ES] Add `Featured Artists` & `Featured Artist showcase beatmaps`

### PR DESCRIPTION
**Pull request by Ladwick**
**Article translation** – `Artistas Destacados` & `Beatmaps de exhibición de Artista Destacado`

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/People/Featured_Artists/es.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  - FA
+---
+
+# Artistas Destacados
+
+Los **Artistas Destacados** (abreviado en inglés como ***FA***) son artistas musicales que han colaborado con osu! para que una selección de sus pistas esté disponible para su uso en [beatmaps](/wiki/Beatmap) y música de fondo en las transmisiones en directo de la [osu! World Cup](/wiki/Tournaments#osu!-world-cup)
+
+El sitio web [listado de Artista Destacado](https://osu.ppy.sh/beatmaps/artists) muestra a cada artista involucrado. La página de cada artista contiene información general acerca de su música y una lista de sus pistas licenciadas para su uso en osu! junto a un archivo `.osz` pre-temporizado para beatmapping.
+
+El [osu! team](/wiki/People/osu!_team) a veces colabora con artistas para crear nuevas pistas llamadas [osu! originals](/wiki/osu!_originals) que se utilizan en osu!. Estas son etiquetadas como tal en el catálogo de Artista Destacado.
```
Comment (UnknownOsu) [inline wiki/People/Featured_Artists/es.md:12]:
```suggestion
El [osu! team](/wiki/People/osu!_team) a veces colabora con artistas para crear nuevas pistas para usar en osu!, llamadas [osu! originals](/wiki/osu!_originals). Estas son etiquetadas como tal en el catálogo de los artistas destacados.
```

File: wiki/People/Featured_Artists/es.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+## Selección
+
+Las adiciones a la librería de Artista Destacado son seleccionadas indirectamente por la comunidad de osu!. Si los mappers crean y clasifican beatmaps con la música de un artista, el osu! team puede ponerse en contacto con dicho artista para discutir el programa de Artista Destacado. Sin embargo, algunos artistas son ocasionalmente invitados a convertirse en Artistas Destacados basándose en otros factores, como su previsible atractivo para la audiencia de mappers de osu!.
```
Comment (UnknownOsu) [inline wiki/People/Featured_Artists/es.md:16]:
```suggestion
Las adiciones a la librería de los artistas destacados son seleccionadas indirectamente por la comunidad de osu!. Si los mappers crean y clasifican beatmaps con la música de un artista, el osu! team puede ponerse en contacto con dicho artista para hablar sobre el programa de artista destacado. Sin embargo, algunos artistas son ocasionalmente invitados a convertirse en artistas destacados basándose en otros factores, como el atractivo que puedan tener para la audiencia de mappers de osu!.
```

File: wiki/People/Featured_Artists/es.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+Todas las canciones del programa de Artista Destacado tienen licencia no exclusiva para su uso en osu!. Más detalles para músicos que deseen unirse a los Artistas Destacados pueden ser encontrados en el artículo de [licenciamiento de música](/wiki/Legal/Music_licensing).
```
Comment (UnknownOsu) [inline wiki/People/Featured_Artists/es.md:18]:
```suggestion
Todas las canciones del programa de artista destacado tienen licencia no exclusiva para su uso en osu!. Más detalles para músicos que deseen unirse a los artistas destacados pueden ser encontrados en el artículo de [licenciamiento de música](/wiki/Legal/Music_licensing).
```

File: wiki/People/Featured_Artists/es.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+## Lanzamientos
+
+Nuevos lanzamientos de Artista Destacado por año:
```
Comment (UnknownOsu) [inline wiki/People/Featured_Artists/es.md:22]:
```suggestion
Nuevos lanzamientos de un artista destacado por año:
```

File: wiki/People/Featured_Artists/es.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+- **2017:** 11
+- **2018:** 12
+- **2019:** 31
+- **2020:** 65
+- **2021:** 101
+- **2022:** 104
+- **2023:** 64
+- **2024:** *por determinar*
+
+Nuevos Artistas Destacados son anunciados semanalmente en el [sitio de noticias](https://osu.ppy.sh/home/news) de osu!. La mayoría de los anuncios incluyen una lista de las canciones licenciadas del artista y un enlace a su página de Artista Destacado, aunque algunos anuncios se emparejan con concursos de mapping o beatmaps creados por el [Mappers' Guild](/wiki/Community/Mappers_Guild).
```
Comment (UnknownOsu) [inline wiki/People/Featured_Artists/es.md:34]:
```suggestion
Los nuevos artistas destacados son anunciados semanalmente en el [sitio de noticias](https://osu.ppy.sh/home/news) de osu!. La mayoría de los anuncios incluyen una lista de las canciones licenciadas del artista y un enlace a su página de artista destacado, aunque algunos anuncios se emparejan con concursos de mapping o beatmaps creados por el [Mappers' Guild](/wiki/Community/Mappers_Guild).
```

File: wiki/People/Featured_Artists/es.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+
+## Uso de la comunidad
+
+Se anima a los beatmappers a crear mapas para las pistas en Artista Destacado. Estas pistas se suministran con datos de [temporización](/wiki/Client/Beatmap_editor/Timing) y ya están autorizadas para su uso en osu!, permitiendo a los usuarios pasar inmediatamente al mapping sin preocupaciones. El cliente en desarrollo de osu!, apodado [osu!(lazer)](/wiki/Client/Release_stream/Lazer), marcará estos mapas prominentemente en el juego.
```
Comment (UnknownOsu) [inline wiki/People/Featured_Artists/es.md:38]:
```suggestion
Se anima a los beatmappers a crear mapas para las pistas de los artistas destacados. Estas pistas se suministran con datos de [temporización](/wiki/Client/Beatmap_editor/Timing) y ya están autorizadas para su uso en osu!, permitiendo a los usuarios pasar inmediatamente al mapping sin preocupaciones. El cliente en desarrollo de osu!, apodado [osu!(lazer)](/wiki/Client/Release_stream/Lazer), marcará estos mapas prominentemente en el juego.
```

File: wiki/People/Featured_Artists/es.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+
+Todos los concursos de mapping publicados en el [listado de concursos de la comunidad](https://osu.ppy.sh/community/contests), tales como el [Monthly Beatmapping Contest](/wiki/Contests/Monthly_Beatmapping_Contest) y el [Mapping Olympiad](/wiki/Contests/Mapping_Olympiad), requieren que los participantes creen mapas con canciones de Artista Destacado. Algunos concursos dirigidos por la comunidad también utilizan estas canciones.
```
Comment (UnknownOsu) [inline wiki/People/Featured_Artists/es.md:40]:
```suggestion
Todos los concursos de mapping publicados en el [listado de concursos de la comunidad](https://osu.ppy.sh/community/contests), tales como el [Monthly Beatmapping Contest](/wiki/Contests/Monthly_Beatmapping_Contest) y el [Mapping Olympiad](/wiki/Contests/Mapping_Olympiad), requieren que los participantes creen mapas con canciones de un artista destacado. Algunos concursos dirigidos por la comunidad también usan estas canciones.
```

File: wiki/People/Featured_Artists/es.md [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+
+Para promocionar el mapping de contenido de Artista Destacado, los miembros de la comunidad crearon el [Mappers' Guild](/wiki/Community/Mappers_Guild), dándole a los mappers un sistema de misiones estilo MMO en el cual pueden participar junto a otros mappers. Los premios se conceden por crear y clasificar beatmaps, incluyendo tres tiers de insignias de perfil. Los mappers pueden unirse al gremio visitando el [sitio web de Mappers' Guild](https://mappersguild.com/). A partir de [Abril de 2023](https://osu.ppy.sh/home/news/2023-04-09-new-featured-artist-chon), los mappers también pueden discutir todo lo relacionado a la creación de beatmaps de Artistas Destacados en el canal `#featured-artists` en el [servidor de Discord de osu!](/wiki/Community/osu!_Discord_server).
```
Comment (UnknownOsu) [inline wiki/People/Featured_Artists/es.md:42]:
```suggestion
Para promocionar el mapping del contenido de los artistas destacados, los miembros de la comunidad crearon el [Mappers' Guild](/wiki/Community/Mappers_Guild), dándole a los mappers un sistema de misiones estilo MMO en el cual pueden participar junto a otros mappers. Los premios se conceden por crear y clasificar beatmaps, incluyendo tres niveles de insignias de perfil. Los mappers pueden unirse al gremio visitando el [sitio web del Mappers' Guild](https://mappersguild.com/). A partir de [abril de 2023](https://osu.ppy.sh/home/news/2023-04-09-new-featured-artist-chon), los mappers también pueden discutir todo lo relacionado a la creación de beatmaps de artistas destacados en el canal `#featured-artists` en el [servidor de Discord osu!](/wiki/Community/osu!_Discord_server).
```

File: wiki/People/Featured_Artists/es.md [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+
+Si un `.osz` pre-temporizado del listado de Artista Destacado tiene errores de cualquier tipo, pueden ser reportados en este [hilo del foro](https://osu.ppy.sh/community/forums/topics/783985).
```
Comment (UnknownOsu) [inline wiki/People/Featured_Artists/es.md:44]:
```suggestion
Si un `.osz` pre-temporizado del listado de los artistas destacados tiene errores de cualquier tipo, pueden ser reportados en este [hilo del foro](https://osu.ppy.sh/community/forums/topics/783985).
```

File: wiki/People/Featured_Artists/es.md [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+
+## Artistas Destacados con cuentas de osu!
```
Comment (UnknownOsu) [inline wiki/People/Featured_Artists/es.md:46]:
```suggestion
## Artistas destacados con cuentas de osu!
```

File: wiki/People/Featured_Artists/es.md [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+
+Los Artistas Destacados con una cuenta de osu! pueden ser identificados por un [título de usuario](/wiki/Community/User_title) cyan de «Artista Destacado» y una insignia de [grupo de usuario](/wiki/People/User_group) que dice `FA` para exhibir su participación.
```
Comment (UnknownOsu) [inline wiki/People/Featured_Artists/es.md:48]:
```suggestion
Los artistas destacados con una cuenta de osu! pueden ser identificados por un [título de usuario](/wiki/Community/User_title) cian «Featured Artist» y una insignia de [grupo de usuario](/wiki/People/User_group) que dice `FA` para exhibir su participación.
```

File: wiki/People/Featured_Artists/es.md [hunk 12]
```diff
@@ -0,0 +1,110 @@
...
+
+La [página de grupo de Artista Destacado](https://osu.ppy.sh/groups/35) enumera todas estas cuentas, mientras que el [listado de Artista Destacado](https://osu.ppy.sh/beatmaps/artists) enumera a todos los Artistas Destacados.
```
Comment (UnknownOsu) [inline wiki/People/Featured_Artists/es.md:50]:
```suggestion
La [página de grupo de los artistas destacados](https://osu.ppy.sh/groups/35) enumera todas estas cuentas, mientras que el [listado de los artistas destacados](https://osu.ppy.sh/beatmaps/artists) enumera a todos los artistas destacados.
```

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]
add Cinamoro

**Review by TicClick — APPROVED** [review]

**Pull request merged**